### PR TITLE
Added custom emojis and disabling buttons when no more pages

### PR DESCRIPTION
**Pull request by BlueTree242**

### CODE REVIEW

File: src/main/examples/Main.java [hunk 1]
```diff
@@ -34,8 +34,8 @@ public static void main(String[] args) throws LoginException {
             .setEmbeds(embeds)
             .setJDA(jda)
             .setChannel(0)
-            .setPredicate(event -> { event.getMember().getIdLong() == 0 })
-            .build()
+            .setPredicate(event -> { return event.getMember().getIdLong() == 0; })
```
Comment (arynxd) [inline src/main/examples/Main.java:37]: this is un-needed

File: src/main/java/me/arynxd/button_pagination/ButtonPaginator.java [hunk 2]
```diff
@@ -80,6 +77,14 @@ public void paginate() {
         doWait();
     }
 
+    public ActionRow getActionRow() {
+        return ActionRow.of(
+                page == maxPage ? Button.primary(jwtTokens.get(1), emojis[1]).asDisabled() : Button.primary(jwtTokens.get(1), emojis[1]).asEnabled(),
```
Comment (arynxd) [inline src/main/java/me/arynxd/button_pagination/ButtonPaginator.java:82]: improve the readability of this

File: src/main/java/me/arynxd/button_pagination/ButtonPaginator.java [hunk 3]
```diff
@@ -118,25 +123,23 @@ private void switchPage(ButtonClickEvent event) {
...
-            case ARROW_RIGHT_UNICODE:
-                page++;
-                if (page >= maxPage) {
-                    page = 0;
-                }
-                break;
-            case WASTEBASKET_UNICODE:
-                message.delete().queue();
-                return;
+        if (emoji.equals(jwtTokens.get(0))) {
```
Comment (arynxd) [inline src/main/java/me/arynxd/button_pagination/ButtonPaginator.java:126]: handle the exceptional else case

File: src/main/java/me/arynxd/button_pagination/ButtonPaginator.java [hunk 4]
```diff
@@ -203,6 +207,16 @@ public Builder setPredicate(Predicate<ButtonClickEvent> predicate) {
             return this;
         }
 
+
+        /**
+         * @param emojis The emojis of the buttons in order: backward, forward, delete
+         * @return The Current Builder
+         */
+        public Builder setEmojis(Emoji... emojis) {
```
Comment (arynxd) [inline src/main/java/me/arynxd/button_pagination/ButtonPaginator.java:215]: this should have a length check

**Review by arynxd — CHANGES_REQUESTED** [review]
also you need to pull upstream

**Comment by arynxd** [comment]
i will resolve the conflicts once you pull upstream & make the required changes

**Comment by BlueTree242** [comment]
Ok i mean alot of problems but ok

**Comment by arynxd** [comment]
once you pull from upstream & make the changes there should be no conflicts

**Pull request closed without merging**